cmd/kvetch/settings.go

```diff
@@ -6,13 +6,91 @@ import (
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/golang/protobuf/ptypes/wrappers"
+	kvstore "github.com/syncromatics/kvetch/internal/datastore"
 )
 
 type settings struct {
 	Port                      int
 	PrometheusPort            int
 	Datastore                 string
 	GarbageCollectionInterval time.Duration
+	KVStoreOptions            *kvstore.KVStoreOptions
+}
+
+func getKVStoreOptions() (*kvstore.KVStoreOptions, error) {
+	allErrors := []string{}
+	kvStoreOptions := &kvstore.KVStoreOptions{}
+	enableTruncateString, ok := os.LookupEnv("ENABLE_TRUNCATE")
```

Comment:
These env variables need to be documented in the README.

Also, any reason not to use Cobra or Viper for configs here instead of direct `os.Lookup` calls? (Examples of Cobra in `kvetchctl`)

Reply:
Just keeping it simple on the env variables, don't see a need to introduce something new there. 

Will add a section to the README for all the environmental variables as that does not exist yet.

Reply:
Cobra's super good for making great CLIs, but since kvetch is more of a long-running service process I don't think we need it

Reply:
:+1: for adding the environment variables to the readme.  otherwise, lgtm.

Reply:
Added a snazzy table for environmental variables in a Configuration section in the readme, can view markup in nice form here....

https://github.com/syncromatics/kvetch/tree/feature/performance-tweaking

Let me know how it looks! I split up the very specific badgerDB settings from the more general kvetch service config. Thanks!

Reply:
looks awesome!